modules/@apostrophecms/ui/ui/apos/components/AposButtonSplit.vue

```diff
@@ -151,27 +156,6 @@ function initialize() {
   setButton(initial);
 }
 
-function trapFocus() {
-  const selectedElementIndex = props.menu.findIndex(i => i.action === action.value) || 0;
-
-  // use map to keep items order:
-  elementsToFocus.value = props.menu.map(
-    ({ action }) => choices.value.find(choice => {
-      return choice.value === action;
-    })
-  );
-
-  focusElement(elementsToFocus.value[selectedElementIndex]);
-}
-function menuOpen() {
-  nextTick(() => {
-    trapFocus();
```

Comment:
Still supports keyboard based selection right?

Reply:
Yes, this was double focus trapping. We do it in the context menu now, with an option for explicit opt-out. There is on reason to opt-out in this case as default behavior is perfectly valid.